sphinx/tutorial/rose/suites.rst

```diff
@@ -0,0 +1,611 @@
+
+   Screenshot of a Rose Bush web page.
+
+If a Rose Bush server is provided at your site, you can open the Rose Bush
+page for a suite by running the :ref:`command-rose-suite-log` command
+in the suite directory.
+
+Otherwise an add-hoc web server can be set up using the
+:ref:`command-rose-bush` ``start`` command argument.
+
+
+.. practical::
+
+   .. rubric:: In this practical we will take the ``forecast`` Rose application
+      that we developed in the :ref:`Metadata Tutorial <tutorial-rose-metadata>`
+      and integrate it into the :ref:`weather-forecasting suite
+      <tutorial-datetime-cycling-practical>`.
+
+   Move into the suite directory from the previous practical::
+
+      cd ~/rose-tutorial/rose-suite-tutorial
+
+   You will find a copy of the ``forecast`` application located in
+   ``app/forecast``.
+
+   #. **Create A Test Configuration For The** ``forecast`` **Application.**
+
+      We have configured the ``forecast`` application to use test data. We will
+      now adjust this configuration to make it work with real data generated by
+      the Cylc suite. It is useful to keep the ability to run the application
+      using test data, so we won't delete this configuration. Instead we
+      will move it into an :ref:`Optional Configuration` so that we can run
+      the application in "test mode" or "live mode".
+
+      Optional configurations are covered in more detail in the
+      :ref:`Optional Configurations
+      Tutorial <rose-tutorial-optional-configurations>`. For now all we need to
+      know is that they enable us to store alternative configurations.
+
+      Create an optional configuration called ``test`` inside the ``forecast``
+      application::
+
+         mkdir app/forecast/opt
+         touch app/forecast/opt/rose-app-test.conf
+
+      This optional configuration is a regular Rose configuration file. Its
+      settings will override those in the :rose:file:`rose-app.conf` file if
+      requested.
+
+      .. tip::
+
+         Take care not to confuse the ``rose-app.conf`` and
+         ``rose-app-test.conf`` files used within this practical.
+
+      Move the following environment variables from the
+      ``app/forecast/rose-app.conf`` file into an ``[env]``
+      section in the ``app/forecast/opt/rose-app-test.conf`` file:
+
+      * ``WEIGHTING``
+      * ``WIND_CYCLES``
+      * ``WIND_FILE_TEMPLATE``
+      * ``RAINFALL_FILE``
+      * ``MAP_FILE``
+      * ``CYLC_TASK_CYCLE_POINT``
+      * ``RESOLUTION``
+      * ``DOMAIN``
+
+      .. spoiler:: Solution warning
+
+         The ``rose-app-test.conf`` file should look like this:
+
+         .. TODO - load this file from the tutorials directory
+
+         .. code-block:: rose
+
+            [env]
+            WEIGHTING=1
+            WIND_CYCLES=0
+            WIND_FILE_TEMPLATE=test-data/wind_{cycle}_{xy}.csv
+            RAINFALL_FILE=test-data/rainfall.csv
+            MAP_FILE=map.html
+            CYLC_TASK_CYCLE_POINT=20171101T0000Z
```

Comment:
Noting for future improvements brief conversation in person regarding setting this variable to ``20171101T0000Z`` being a potential cause of confusion for readers, given it is a random date-time tied to the test data.

Reply:
I've inserted a better explanation of the test data in a42675a.